docker-compose.yml

```diff
@@ -126,6 +126,7 @@ services:
             APP_DEBUG: ${APP_DEBUG:-0}
         env_file:
             - ./api/api.env
+            - ./api/tests/Behat/test.env
```

Comment:
Is this needed as, aren't you still using the test container for tests but just with the context of API? Or maybe these env vars are needed for API container to run?

Reply:
I had originally dropped this but added them back in desperation during the file scan fiasco. I'll experiment with dropping them both out.

Reply:
Yep works fine - removed.